netmap/netmap_contract.go

```diff
@@ -52,9 +55,10 @@ const (
 
 const (
 	// V2 format
```

Comment:
hm, that is not a `V2 format` then if "For historical reasons numerical values differs from the ones described in the NeoFS API."?

Reply:
Fixed